closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -43,8 +44,9 @@ else ifeq ($(call isTargetOs, linux), true)
   endif
 else ifeq ($(call isTargetOs, windows), true)
   ifeq ($(call isTargetCpu, x86_64), true)
-    OPENJCEPLUS_BOOT_JDK := $(call MixedPath,$(OPENJCEPLUS_BOOT_JDK))
+    OPENJCEPLUS_JDK := $(call MixedPath,$(OPENJCEPLUS_JDK))
     OPENJCEPLUS_GSKIT_HOME := $(call MixedPath,$(OPENJCEPLUS_GSKIT_HOME))
+    OPENJCEPLUS_HEADER_FILES := $(call MixedPath,$(SUPPORT_OUTPUTDIR)/headers/openjceplus)
```

Comment:
This should use `OPENJCEPLUS_HEADER_FILES` defined above:
```
    OPENJCEPLUS_HEADER_FILES := $(call MixedPath,$(OPENJCEPLUS_HEADER_FILES))
```

Reply:
Fixed.